src/components/pacbio/PacbioPoolEdit.vue

```diff
@@ -163,13 +161,14 @@ export default {
         this.parsedFile = false
       }
     },
+    onFieldUpdate() {
+      this.validated = false
+    },
```

Comment:
Maybe, comment here as well  on what it does would be helpful  for future support

Reply:
Yes, done